### PR TITLE
[AIFN] Add new object type AIFN

### PR DESCRIPTION
**Pull request by D047539**

### CODE REVIEW

File: file-formats/aifn/type/zif_aff_aifn_v1.intf.abap [hunk 1]
```diff
@@ -0,0 +1,27 @@
+INTERFACE zif_aff_aifn_v1
+  PUBLIC.
+  TYPES:
+    "! <p class="shorttext">Namespace Details</p>
+    "! Namespace details
+    BEGIN OF ty_namespace_details,
+      "! <p class="shorttext">AIF Namespace</p>
+      "! AIF Namespace
```
Comment (Markus1812) [inline file-formats/aifn/type/zif_aff_aifn_v1.intf.abap:8]:
```suggestion
      "! AIF namespace
```

Please use sentence case in descriptions

File: file-formats/aifn/type/zif_aff_aifn_v1.intf.abap [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      "! $required
+      aif_namespace TYPE c LENGTH 6,
+    END OF ty_namespace_details,
+
+    "! <p class="shorttext">Recipient</p>
+    "! Recipient
```
[the remark this answers is not in the document]
Comment (D047539) [reply file-formats/aifn/type/zif_aff_aifn_v1.intf.abap]: thanks for the hint, this was a copy and replace issue.

File: file-formats/aifn/type/zif_aff_aifn_v1.intf.abap (hunk 1 above)
Comment (D047539) [reply file-formats/aifn/type/zif_aff_aifn_v1.intf.abap]: thanks, description was adapted

**Review by Markus1812 — APPROVED** [review]
Thanks for the changes, looks good to me. In the next step, the ux-review team will take over.

**Review by albertmink — APPROVED** [review]
Cross-check done. LGTM

**Review by albertmink — APPROVED** [review]
Missed to approve it, so here we go

**Pull request merged**